src/infrastructure/metrics.py

```
@@ -0,0 +1,229 @@
+        metric_name = f"{self.namespace}_{name}"
+        tags_str = self._tags_to_string(tags)
+        key = f"{metric_name}{tags_str}"
+        
+        self._histograms[key].append(value)
+        
+        values = self._histograms[key][-1000:]
+        self._histograms[key] = values
+        
+        if values:
+            sorted_values = sorted(values)
+            count = len(sorted_values)
```

Ruff (0.12.2)

Local variable `count` is assigned to but never used

Remove assignment to unused variable `count`

(F841)